committee-steering/governance/annual-reports.md

```diff
@@ -109,15 +113,17 @@ in any area now? What are you doing about it?
 What are some activities that the group does to encourage this? What programs
 are you participating in to grow contributors throughout the contributor ladder?
 - What programs do you participate in for new contributors?
-- Does the group have contributors from multiple companies/affiliations?
+- Does the group have contributors from multiple companies/affiliations? Can end
+ users/companies contribute in some way that they currently are not?
 
 ##### Current initiatives and project health
 - What are initiatives that should be highlighted, lauded, shout outs, that
 your group is proud of? Currently underway? What are some of the longer tail
 projects that your group is working on?
-- What areas does the group need the most help with?
-- Whats the average open days of a PR and Issue in your group? / what metrics does
- your group care about and/or measure?   
+- Year to date KEP work: What's now stable? Beta? Alpha? Road to alpha?
```

Comment:
```suggestion
- Year to date KEP or initiative work: What's now stable? Beta? Alpha? or rolled out some other method?
```
There might be better wording, but just thinking about trying to capture other initiatives that might not be tracked in KEP form. Some of the cross-cutting work by contribex/arch/testing I think would fall under that category.

Reply:
wdyt about this under the KEP q:
> What initiatives are you working on that aren't being tracked in KEPs?

Reply:
SGTM